# possible values
bool ON
bool OFF

# fields
bool play_pause
bool stop
bool custom